cookbooks/bcpc-hadoop/recipes/oozie.rb

```diff
@@ -214,8 +214,30 @@
 ruby_block 'update sharelib checksum' do
   block do
     require 'digest'
-    node.set[:bcpc][:hadoop][:oozie][:sharelib_checksum] =
-      Digest::MD5.hexdigest(File.read(OOZIE_SHARELIB_TARBALL_PATH))
+    node[:bcpc][:hadoop][:oozie_hosts].each do |oozie_host|
```

Comment:
So all Oozie hosts connect to all Oozie hosts?

Reply:
Yes it always does. https://github.com/bloomberg/chef-bach/blob/master/cookbooks/bcpc-hadoop/recipes/oozie.rb#L226

Reply:
Discussed with @amithkanand that oozie should be up and running before generating the sharelib. It's probably better to rearrange the sequence of converges